indigo_app/views/places.py

```diff
@@ -1079,22 +1088,99 @@ def get_context_data(self, **kwargs):
 
         work = self.object
 
-        context["documents"] = Document.objects.undeleted().no_xml().filter(work=work)
-        context["tab"] = "docs" if context["documents"] else None
+        context["overview_data"] = self.get_overview_data()
+        context["tab"] = "overview"
 
-        # count commenced works
+        # count documents
+        context["n_documents"] = Document.objects.undeleted().filter(work=work).count()
+
+        # count commencements
+        context["n_commencements"] = work.commencements.count()
         context["n_commencements_made"] = work.commencements_made.count()
-        context["n_commenced_by"] = work.commencements.filter(commencing_work__isnull=False).count()
-        context["tab"] = context["tab"] or (
-            "commencements" if context["n_commencements_made"] + context["n_commenced_by"] else None
-        )
 
-        # count amended works
-        context["n_amendments_made"] = work.amendments_made.count()
+        # count amendments
         context["n_amendments"] = work.amendments.count()
-        context["tab"] = context["tab"] or (
-            "amendments" if context["n_amendments_made"] + context["n_amendments"] else None
-        )
+        context["n_amendments_made"] = work.amendments_made.count()
+
+        # count repeals
+        context["n_repeals"] = 1 if work.repealed_by else 0
+        context["n_repeals_made"] = work.repealed_works.count()
+
+        # count primary / subsidiary works
+        context["n_primary_works"] = 1 if work.parent_work else 0
+        context["n_subsidiary_works"] = work.child_works.count()
+
+        # count tasks
+        context["n_tasks"] = work.tasks.filter(state__in=Task.OPEN_STATES).count()
+
+        return context
+
+    def get_overview_data(self):
+        """ Return overview data for the work as a list of OverviewDataEntry objects"""
+        # TODO: are the translations being done correctly here?
+        def format_date(date_obj):
```

Comment:
I don't think you should do this as part of this PR. I deliberately have used YYYY-MM-DD as far as possible to avoid having to localise. We can do it later if we need to.